src/daemon/config.rs

```diff
@@ -544,6 +545,38 @@ impl Config {
         }
     }
 
+    pub fn requeue_remote_failed(&self) -> Priority {
+        if test_mode_enabled() {
+            in_seconds(5)
+        } else {
+            in_seconds(SCHEDULER_REQUEUE_DELAY_SECONDS)
+        }
+    }
+
+    /// Get the priority for the next CA refresh based on the configured
+    /// ca_refresh_seconds (1 day), and jitter (12 hours)
+    pub fn ca_refresh_next(&self) -> Priority {
+        Self::ca_refresh_next_from(self.ca_refresh_seconds, self.ca_refresh_jitter_seconds)
+    }
+
+    pub fn ca_refresh_start_up(&self, use_jitter: bool) -> Priority {
+        let jitter_seconds = if use_jitter { self.ca_refresh_jitter_seconds } else { 0 };
+
+        Self::ca_refresh_next_from(0, jitter_seconds)
+    }
+
+    fn ca_refresh_next_from(regular_seconds: i64, jitter_seconds: i64) -> Priority {
```

Comment:
Should `jitter_seconds` be `u64`? The 0..n range expression won't work if it is negative, right?

Reply:
yes.. probably.. I created issue #808 for this. This affects more settings that we should review.